tests/Export/XmlExportTest.php

```diff
@@ -77,6 +77,15 @@ public function testProductsInCrossSellCategoriesAreNotWrappedAndErrorIsLogged()
         );
     }
 
+    public function testKeywordsAreNotRequired(): void
+    {
+        $product = $this->createVisibleTestProduct(['tags' => []]);
+        $items = $this->getExport()->buildItems([$product], self::VALID_SHOPKEY, []);
+
+        $this->assertCount(1, $items);
+        $this->assertSame($product->getId(), $items[0]->getId());
+    }
```

Comment:
Please also assert that the keywords are empty.

Reply:
I tried doing this, but libflexport does not provide any way to check this.

![image](https://user-images.githubusercontent.com/20555112/98655030-4f94dd80-233f-11eb-89f6-29df4798f0a2.png)

I would need to generate the item with an exporter and then manually parse that XML string.

Reply:
No `getKeywords`? Libflexport would need a improvement :) Let's keep it at the id then.